src/OpenTK/Platform/X11/API.cs

```diff
@@ -52,6 +52,13 @@ namespace OpenTK.Platform.X11
 
     #endregion
 
+    // X11 has some defined values
+    internal static class Consts
+    {
+        public static readonly IntPtr None = IntPtr.Zero;
+        public static readonly IntPtr CurrentTime = IntPtr.Zero;
+    }
+
```

Comment:
Wow. Why on earth is that defined this way? If you do know, just note it in the comment. If you don't, then just explain where you got it from.